workshop/1-Personalization/1.2-Real-Time-Events-Segment.ipynb

```diff
@@ -0,0 +1,451 @@
+    "        Name='retaildemostore-segment-write-key',\n",
+    "        Value='{}'.format(segment_write_key),\n",
+    "        Type='String',\n",
+    "        Overwrite=True\n",
+    "    )\n",
+    "\n",
+    "print(segment_write_key)"
+   ]
+  },
+  {
+   "source": [
+    "You now have an environment variable that will enable the Segment data collection library in the code in `index.html`.  All we need to do now, is force a re-deploy of the Retail Demo Store.  \n",
+    "\n",
+    "To do that, go back to your AWS Console tab or window, and select Code Pipeline from the Services search.  Then, find the pipeline name that contains `WebUIPipeline` and click that link.\n",
+    "\n",
+    "![Segment Destination Setup](images/segment/segment-step-25.png)\n",
+    "\n",
+    "Then, select the ‘Release Change’ button, and confirm the release once the popup shows up.  You will see a confirmation that the pipeline is re-deploying the web ui for the Retail Demo Store.\n",
+    "\n",
+    "![Segment Destination Setup](images/segment/segment-step-26.png)\n",
+    "\n",
+    "This process should complete in a few minutes.  Once this is done, you will see the bottom tile confirm that your deployment has completed:\n",
+    "\n",
+    "![Segment Destination Setup](images/segment/segment-step-27.png)\n",
+    "\n",
+    "## Log in as a Retail Demo Store User\n",
+    "\n",
+    "Once this is confirmed, tab back to the Retail Demo Store web UI, and refresh the screen to reload the user interface.  In the Personalize workshop, you created an account for the Retail Demo Store.  If you are not logged in as that account, log in now.\n",
+    "\n",
+    "Then, open another tab to the Segment console, and select ‘retail-demo-store’ under the Sources tab:\n",
+    "\n",
+    "![Segment Destination Setup](images/segment/segment-step-28.png)\n",
+    "\n",
+    "And then click on the Debugger tab.\n",
+    "\n",
+    "![Segment Destination Setup](images/segment/segment-step-29.png)\n",
+    "\n",
+    "Here you will be able to see live events coming from the Retail Demo Store this:\n",
+    "\n",
+    "![Segment Destination Setup](images/segment/segment-step-30.png)\n",
+    "\n",
+    "Go back to the Retail Demo Store web app tab, and click on some categories of products.  You should start to see events appear in the Segment debugger at this point.  This means that the actions you are performing as a user are now being sent to the training datasets in Personalize.\n",
+    "\n",
+    "Now that you have events flowing from your web application to Personalize, you can keep the Personalize data sets up to date, and re-train them as required by your application use cases.\n",
+    "\n",
+    "[ TODO:  Show events flowing in the Cloud Watch logs? ]\n",
+    "\n",
+    "You can finish the workshop here, or learn more about what you can do with Segment’s Customer Data Platform (Personas) and Amazon Personalize in the next workbook:\n",
```

Comment:
The CDP workshop is another directory. I suggest hyperlinking to the notebook so it's easier for the attendee to jump over to it.